cmd/utils/hierarchical_coordinator.go

```diff
@@ -779,18 +779,21 @@ search:
 	nodeSet := NodeSet{nodes: make(map[string]Node)}
 	primeNode, err := hc.NodeFromHash(modifiedConstraintMap[common.Location{}.Name()], common.Location{})
 	if err != nil {
+		log.Global.Error("error reading node from hash in prime", "err ", err)
 		return
 	}
 	nodeSet.nodes[common.Location{}.Name()] = primeNode
 	for i := 0; i < int(numRegions); i++ {
 		regionNode, err := hc.NodeFromHash(modifiedConstraintMap[common.Location{byte(i)}.Name()], common.Location{byte(i)})
 		if err != nil {
+			log.Global.Error("error reading node from hash in region ", i, "err ", err)
 			return
 		}
 		nodeSet.nodes[common.Location{byte(i)}.Name()] = regionNode
 		for j := 0; j < int(numZones); j++ {
 			zoneNode, err := hc.NodeFromHash(modifiedConstraintMap[common.Location{byte(i), byte(j)}.Name()], common.Location{byte(i), byte(j)})
 			if err != nil {
+				log.Global.Error("error reading node from hash in zone ", common.Location{byte(i), byte(j)}, "err ", err)
```

Comment:
The proper way to do these is WithField(s)

Reply:
cool, i can change, have any other comments ?

Reply:
Fixed this and all the other logs in hierarchical coordinator